### PR TITLE
[Keyboard] Add xox70

### PR DESCRIPTION
**Pull request by rhmokey**
## Description
Add new keyboard xox70

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mokey/xox70/xox70.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+) \
+{ \
+    { k00,  k01,   k02,   k03,  k04,  k05,   k06,  k07,   k08,  k09,   k0a,  k0b,  k0c,  KC_NO, k0e,     k0f,   k0g,   k0h }, \
+    { k10,  KC_NO, k12,   k13,  k14,  k15,   k16,  k17,   k18,  k19,   k1a,  k1b,  k1c,   k1d,  k1e,     k1f,   k1g,   k1h }, \
+    { k20,  KC_NO, k22,   k23,  k24,  k25,   k26,  k27,   k28,  k29,   k2a,  k2b,  k2c,   k2d,  KC_NO,   KC_NO, KC_NO,KC_NO }, \
+    { k30,  KC_NO, k32,   k33,  k34,  k35,   k36,  k37,   k38,  k39,   k3a,  k3b,  KC_NO, k3d,  KC_NO,   KC_NO,  k3g, KC_NO }, \
+    { k40,  k41,   KC_NO, k43, KC_NO, KC_NO, k46, KC_NO, KC_NO, KC_NO, k4a,  k4b,  KC_NO, k4d,  k4e,      k4f,   k4g,   k4h }  \
+}
+
+#define LAYOUT_all(\
```
Comment (drashna) [inline keyboards/mokey/xox70/xox70.h:37]: why two `layout_all`'s?

Comment (rhmokey) [reply keyboards/mokey/xox70/xox70.h:37]: One is "layout_all", the other is "layout_all_1", which means there are two levels of changes. Shouldn't it be named like this? If not, I can correct it

File: keyboards/mokey/xox70/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# xox70 PCB
+
+![xox70](https://raw.githubusercontent.com/rhmokey/123.github.io/main/2021/08/15/hello-world/XOX70.jpg)
+
+Support Atmega32u4 keyboard,With a variety of configurations
+
+* Keyboard Maintainer: [Mokey](https://github.com/rhmokey)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/mokey/xox70/readme.md:9]: This needs to be filled out.

File: keyboards/mokey/xox70/xox70.h (hunk 1 above)
Comment (drashna) [reply keyboards/mokey/xox70/xox70.h:37]: `LAYOUT_all` should have all matrix positions available for the board.  So ... you shouldn't need or have 2 LAYOUT_all macros.

**Pull request closed without merging**